resources/views/components/form-field.blade.php

```diff
@@ -0,0 +1,39 @@
+{{--
+@props(['type' => 'info', 'message'])
+<code>x-form-field</code>
+--}}
+<div class="{{ ($inline ?? false) ? 'lg:flex' : '' }} {{ ($fullWidth ?? false) ? '' : '' }} mb-2">
+    @if($label ?? false)
+        <div class="{{ ($fullWidth ?? false) ? '' : (($inline ?? false) ? 'lg:w-36 lg:pr-3 lg:text-right' : '') }} pt-1 whitespace-nowrap {{ $errors && $errors->has($attribute ?? null) ? 'text-danger' : '' }}">
+            <label for="{{ $fieldId ?? $attribute ?? null }}">
+                {{ $label }} {{ !empty($required) ? '*' : '' }}
+            </label>
+        </div>
+    @endif
+    <div class="grow">
+        @if(!empty($icon))
+            <div class="input-group">
+                <span class="input-group-prepend">
+                    <span class="input-group-text">
+                        {{ svg('fas-'.$icon, 'icon') }}
+                    </span>
+                </span>
+                {{ $slot }}
+            </div>
+        @else
+            {{ $slot }}
+        @endif
+        @if(!empty($help))
+            <p class="help-block text-secondary mb-0">
+                {!! $help  !!}
+            </p>
+        @endif
+        @if($attribute ?? null)
+            @error($attribute)
+            <p class="help-block text-danger mb-0">
+                <x-fas-exclamation-triangle /> {{ $errors->first($attribute) }}
+            </p>
```

Comment:
When these are visible, is there a good way for the input field to have an `aria-describedby` attribute that points to their IDs?

Reply:
Phew, that's a neat detail. Addressed in https://github.com/RetroAchievements/RAWeb/pull/1831/commits/12f89de8f7c536e07e7de6f667e008c11be29367